packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -0,0 +1,311 @@
+    "issuer": "https://${yourHubOrgDomain}"
+}' "https://${yourHubOrgDomain}/api/v1/apps/${yourServiceAppId}/grant"
+```
+
+```bash
+curl -X POST \
+  -H 'Accept: application/json' \
+  -H "Authorization: SSWS ${api_token}" \
+  -H 'Content-Type: application/json' \
+  -d ' {
+    "scopeId": "okta.groups.manage",
+    "issuer": "https://${yourHubOrgDomain}"
+}' "https://${yourHubOrgDomain}/api/v1/apps/${yourServiceAppId}/grant"
+```
+
+### Enable provisioning in the Org2Org app
+
+In each spoke org, set and activate provisioning for the Org2Org app integration by using the [Okta Apps API](/docs/reference/api/apps/#set-default-provisioning-connection-for-application).
+
+> **Note**: Currently, only the Okta API can be used to enable OAuth 2.0-based provisioning.
+
+As an Okta admin, make a [`POST /api/v1/apps/${Org2OrgAppId}/connections/default?activate=TRUE`](/docs/reference/api/apps/#set-default-provisioning-connection-for-application) request to set provisioning for the spoke org using the following profile parameters:
+
+| Parameter |  Description/Value   |
+| --------- |  ------------- |
+| `authScheme`  |  `OAUTH2` |
+| `clientId`  |  Specify the corresponding service app client ID in your hub org. |
+
+##### Request example
+
+```bash
+curl -X POST \
+  -H 'Accept: application/json' \
+  -H "Authorization: SSWS ${api_token}" \
+  -H 'Content-Type: application/json' \
+  -d ' {
+    "profile": {
+        "authScheme": "OAUTH2",
+        "clientId": "${yourHubOrgServiceAppId}"
+    }
+}' "https://${yourSpokeOrgDomain}/api/v1/apps/${yourOrg2OrgAppId}/connections/default?activate=TRUE"
+```
+
+> **Note**: After provisioning is enabled, if you want to edit Org2Org attribute mappings, you can use the [App API features operation](/docs/reference/api/apps/#list-features-for-application) and the [Mappings API](/docs/reference/api/mappings/), or alternatively, you can go to the Okta Admin Console and edit the **Okta Org2Org Attribute Mappings** section of the Org2Org app **Provisioning To App** settings.
+
+### Assign users and groups in the Org2Org app
+
+In each spoke org, assign the users and groups to the Org2Org app integration by using the [Okta Apps API application operations](/docs/reference/api/apps/#application-user-operations):
+
+* [`POST /api/v1/apps/${yourOrg2OrgAppId}/users` (assign user to an application for SSO and provisioning)](/docs/reference/api/apps/#assign-user-to-application-for-sso-and-provisioning)
+* [`POST /api/v1/apps/${yourOrg2OrgAppId}/groups/${groupId}` (assign group to an application)](/docs/reference/api/apps/#assign-group-to-application)
+
+Alternatively, you can assign users and groups for provisioning using the Okta Admin Console. See [ Assign an app integration to a user](https://help.okta.com/oie/en-us/Content/Topics/Provisioning/lcm/lcm-assign-app-user.htm) and [Assign an app integration to a group](https://help.okta.com/oie/en-us/Content/Topics/Provisioning/lcm/lcm-assign-app-groups.htm).
```

Comment:
Not sure if it matters, but might want to remove the space after the [ opening bracket.